liquidctl/driver/kraken3.py

```diff
@@ -186,66 +203,69 @@ def initialize(self, direct_access=False, **kwargs):
 
         # initialize
         if self._hwmon and not direct_access:
-            _LOGGER.info('bound to %s kernel driver, assuming it is already initialized',
-                         self._hwmon.module)
+            _LOGGER.info(
+                "bound to %s kernel driver, assuming it is already initialized", self._hwmon.module
+            )
         else:
             if self._hwmon:
-                _LOGGER.warning('forcing re-initialization despite %s kernel driver',
-                                self._hwmon.module)
-            update_interval = (lambda secs: 1 + round((secs - .5) / .25))(.5)  # see issue #128
-            self._write([0x70, 0x02, 0x01, 0xb8, update_interval])
+                _LOGGER.warning(
+                    "forcing re-initialization despite %s kernel driver", self._hwmon.module
+                )
+            update_interval = (lambda secs: 1 + round((secs - 0.5) / 0.25))(0.5)  # see issue #128
+            self._write([0x70, 0x02, 0x01, 0xB8, update_interval])
             self._write([0x70, 0x01])
 
-        status = []
+        self.status = []
```

Comment:
I can't see the reason for having this extra state, can you explain your logic here?  Also, at the very least it shouldn't be public (according to Python naming conventions).